schema/windows-terminal-quake.schema.1.json

```diff
@@ -14,6 +14,16 @@
 			"default": false
 		},
 
+		"CloseTerminalOnExit": {
+			"$id": "#/properties/CloseTerminalOnExit",
+
+			"title": "Close terminal on exit",
+			"description": "Whether to close the Windows Terminal process when the Windows Terminal Quake process exits.",
```

Comment:
@webbertakken Good point, the main reasons I includes this:
- During development, I often stop and start wt-quake, keeping windows-terminal up in between
- I worry about changing the default behavior, which used to be "keep open"

Either of these are not that great of a reason, so perhaps it would be nicer to have a more obvious (but fixed) default vault.

Would also cut out some complexity, as you suggested.

Reply:
To me, I use the shortcut to wt-quake to start the terminal. So I never directly interact with windows terminal anymore. In my use-case I wouldn't want the terminal to "keep open". 

Currently it does in fact not keep the window open. It closes the tabs.

To me the current behaviour, it being "closing terminal when closing wt-quake console" feels right and should probably stay the default. 

But I guess if you feel the opposite, we might indeed need an option for it, hehe.

Reply:
@webbertakken Well, I thought about it a bit more, and I think that reducing the complexity a bit seems more important than the weird use cases. Especially when doing later changes and needing to re-test everything, since I've notice that restoring the terminal when wt-quake closes is a bit of a pain.

Thank you for your takes on this :)